korman/exporter/mesh.py

```diff
@@ -218,6 +218,11 @@ def _create_geospan(self, bo, mesh, bm, hsgmatKey):
         geospan = plGeometrySpan()
         geospan.material = hsgmatKey
 
+        # Mark us as needing a BlendSpan if the material require blending
+        blend_span = bool(hsgmatKey.object.layers[0].object.state.blendFlags & hsGMatState.kBlendMask)
+        if blend_span:
```

Comment:
Might be best to just combine this all into one line and drop the explicit conversion to a bool.